src/App.js

```diff
@@ -38,14 +38,15 @@ class App extends Component {
         try {
           let feed = await parser.parseURL(CORS_PROXY + feed_url);
           console.log("feed: " + JSON.stringify(feed, null, 4));
-          this.setState({
+          this.setState(state => ({
+            ...state,
             episodes: feed.items,
             program_title: feed.title,
             fetching: !this.state.fetching,
```

Comment:
Surely `fetching: !state.fetching` is preferable if we're using the `state =>` syntax, right?